tests/test_randvars/test_random_variable.py

```diff
@@ -20,7 +20,7 @@ def setUp(self) -> None:
         np.random.seed(42)
 
         # Random variable instantiation
-        self.scalars = [0, int(1), 0.1, np.nan, np.inf]
+        self.scalars = [0, int(1), 0.1]
```

Comment:
the nan and inf screwed with the test_scalar_mult below (warnings were raised). The warnings bothered me slightly, and I didnt see the point of this piece of code, but may well have missed something. This removal will be easy to undo in any case.

Reply:
I don't think I wrote the test, but from my point of view it makes sense to keep it.
As random variables should behave exactly like numpy arrays, they should also mimic their behavior when being multiplied by `np.nan` and `np.inf`.

As for the warnings, we can even assert they're there https://docs.pytest.org/en/stable/warnings.html#warns.

Reply:
yes, I agree that they are worth testing, but not the way they are currently tested (by calling it and getting a bunch of warnings as outputs that need to be read by a human). I will go ahead and write a specialised test for this instead.